internal/flink/command_artifact_create.go

```diff
@@ -9,6 +9,7 @@ import (
 
 	connectcustompluginv1 "github.com/confluentinc/ccloud-sdk-go-v2/connect-custom-plugin/v1"
 
+	"github.com/confluentinc/cli/v3/internal/connect"
```

Comment:
The internal flink package should not be calling the internal connect package. Let's move the `UploadFile()` function to a package within the `pkg/` directory so we can avoid import cycles in the future.

Reply:
done